src/commands/new.ts

```diff
@@ -24,6 +46,10 @@ export default class New extends Command {
   static args = []
 
   async run() {
+		const exampleDescription = await getExamplesFlagDescription();
+		this.flags.example = Flags.string({ char: 'e', description: exampleDescription});
```

Comment:
Okay, so it's been a while now that I'm stuck on this.
The problem that I'm facing is that I fetched the example.json, mapped it, and made the description out of it. But, how do I add to my `static flags`? 
I've tried to add it inside the run() function, before parsing the `New` class, but it's not working.

Reply:
I think flag building doesn't like async/await and promises 😄 

when I quickly play with your code:
- replace any async/await with normal functions and switch to `readFileSync`
- fix building description like this:
  ```
  let description = 'name of the example to use. Available examples are:';
 
  for (const element of examples) {
    description += (`\n\t - ${element.value}`);
  }

it works as expected:
<img width="959" alt="Screenshot 2022-07-19 at 14 17 01" src="https://user-images.githubusercontent.com/6995927/179748224-8b68ef39-2eb2-4e12-af2e-098c5eba247d.png">


full code:

```
function loadExampleFile() : Example[] {
  const exampleFiles = readFileSync(resolve(__dirname, '../../assets/examples/examples.json'), { encoding: 'utf8' });
  //console.log(JSON.parse(exampleFiles));
  return JSON.parse(exampleFiles);
}

function getExamplesFlagDescription() : string {
  const examples = loadExampleFile();
  let description = 'name of the example to use. Available examples are:';
 
  for (const element of examples) {
    description += (`\n\t - ${element.value}`);
  }

  return description;
}
```

```
    example: Flags.string({ char: 'e', description: getExamplesFlagDescription()}),
```

Reply:
Got it, thanks a lot